docs/processes/documentation/documentation.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+Documentation is one of the most important parts of any software epic. It is imperative to remember, you will not be working on the capstone projects forever and that there will be people taking your place within Thoth Tech.
+
+Documentation is the offical information that is provided by the company for users and contributors. Documentation should provide clarity around troubleshooting and provide users and those involved with a basic idea of what it is the platform does and how it works.
+
+The Thoth Tech documentation should all be stored in the documentation repositry within the company GitHub. All documentation should be stored in a .md format and anything to do with research, meeting minutes and updates, should be stored within the [documentation repositry](https://github.com/thoth-tech/documentation).
+
+## Why is it important?
+
+Without documentation, there is no record of:
+
+- Choices and Decisions
+- Why functionality is added
+- What processes are
+- How users use the tool
+- What project updates have been added
+- And more
+
+Thoth Tech documentation are not only used by the one respective team. The documentation can be accesed at anytime by any of the Thoth Tech company members just to see where a project is heading and what progress has been made. Additionally, should other teams wish, they may collaborate and think of intuitve ways in which to further improve the projects.
```

Comment:
I personally would not include the section about collaboration, I do not think this is the right place to suggest such activity and will depend on the area leads of that tri. Better to leave that go future area leads and have one source of truth.

```suggestion
Thoth Tech documentation is accessible to all.
```